assets/components/stripePopUpButton/stripePopUpButton.jsx

```diff
@@ -37,7 +38,7 @@ const StripePopUpButton = (props: PropTypes) => {
     <button
       className="component-stripe-pop-up-button"
       onClick={stripeClick}
-    >Contribute with debit/credit card</button>
+    >Contribute with debit/credit card <Svg svgName="credit-card" /></button>
```

Comment:
By convention, the name of the modules in React are written with the first letter capitalised. This convention looks ok in all the modules that are not a collection of initials.